src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/SolutionExplorerInProcess.cs

```diff
@@ -433,18 +433,6 @@ public async Task AddFileAsync(string projectName, string fileName, string? cont
         }
 
         public async Task RenameFileAsync(string projectName, string oldFileName, string newFileName, CancellationToken cancellationToken)
-        {
-            await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
```

Comment:
See first commit: this VsShellUtilities.RenameDocument helper only updates the RDT but doesn't actually rename the file on disk or anything useful like that. I think the method is intended to be a helper for somebody implementing the entire end-to-end rename, but isn't useful in its own right.